src/htss/plans/detector.py

```diff
@@ -64,15 +64,14 @@ def set_roi(det: AdAravisDetector, roi: Roi) -> Generator:
     Yields:
         Plan
     """
-
-    max_x = yield from bps.rd(det.cam.max_size.max_size_x)
-    max_y = yield from bps.rd(det.cam.max_size.max_size_y)
+    #Ophyd Async AravisDetector doesn't appear to have signal for max sizes
```

Comment:
Do you know if we should be looking at max sizes still?

Reply:
Hmm, we may need to add that signal to the base detector in ophyd-async